### PR TITLE
IPv6 and ICMPv6 support added to check_icmp 

### PR DESCRIPTION
**Pull request by box293**
This resolves issue https://github.com/nagios-plugins/nagios-plugins/issues/143 and https://github.com/nagios-plugins/nagios-plugins/issues/322.

### CODE REVIEW

File: plugins-root/check_icmp.c [hunk 1]
```diff
@@ -209,12 +227,13 @@ static int wait_for_reply(int, u_int);
 static int recvfrom_wto(int, void *, unsigned int, struct sockaddr *, u_int *, struct timeval*);
 static int send_icmp_ping(int, struct rta_host *);
 static int get_threshold(char *str, threshold *th);
+/* investigate */
```
Comment (mjtrangoni) [inline plugins-root/check_icmp.c]: @box293 I think you can remove this line

**Comment by mjtrangoni** [comment]
@box293 Did you try running `clang-format`? I think this should be run once at least.

**Comment by box293** [comment]
@mjtrangoni thanks for the feedback, I've done just that :)

**Comment by hedenface** [comment]
Closes #143.

Closes #322.

**Pull request merged**